__tests__/api.test.js

```diff
@@ -68,6 +68,24 @@ describe("4. GET /api/articles/:article_id", () => {
         expect(body.article).toMatchObject(objCorrect);
       });
   });
+  test("should respond with object when obj with 0 comments if article has 0 comments", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .then(({ body }) => {
+        const objCorrect = {
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: 2,
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          comment_count: 0,
+        };
+
+        expect(body.article).toMatchObject(objCorrect);
+      });
+  });
```

Comment:
this test should be in the /api/articles/2/comments endpoint - it should be a status 200 and return and empty array when trying to access an article with no comments